BootloaderCommonPkg/Library/IppCryptoLib/rsa_verify.c

```diff
@@ -54,6 +56,10 @@ int VerifyRsaSignature (CONST PUB_KEY_HDR *PubKeyHdr, CONST SIGNATURE_HDR *Signa
     return err;
   }
 
+  // Allocate BN Buf of size 6K bytes
+  bn_buf_sz = 6*1024;
+  bn_buf = AllocatePool (bn_buf_sz);

```

Comment:
Please check NULL return. No need to do ZeroMem?

Reply:
also there is API available to get the required buffer size in IPP.
Please try to use API to get it and then allocate it.